crates/runner-host/crates/web-common/src/lib.rs

```diff
@@ -51,6 +51,10 @@ pub enum Command {
 
     /// Prints a debug message.
     Log { message: String },
+
+    // Input from user to console.
+    #[serde(rename_all = "camelCase")]
+    Input { message: String },
```

Comment:
We don't support console input yet (see #236), but I guess you added this for testing purposes. We could indeed just print those in the host runner with a warning that it was ignored.

Reply:
I still don't understand why you added input support. Where do you need it? I would suggest simply removing console input in the Web UI completely. This is misleading since users can't input anything from the Web UI. The current UI is already misleading which is probably why you added it. In the current UI I don't think anything is sent over web-socket. This is purely something on JS side.

So I would actually suggest removing this to avoid further misunderstanding of what is supported.

Reply:
The reason was I wanted to exactly match the behavior of the current web client. 
Currently if you observe the websocket activity of the current client on console input it sends the following json to the backend:
```
{
type: "input"
message: "<Message contents>"
}
```

I'd have no problem removing it for now though. Just with the caveat that it will now behave differently than the current web client. 

If that sounds good to you I'll go ahead and remove it.

Reply:
Yes, I think it's a bug of the current UI, not a feature. We should remove this.

Reply:
Alright sg. I have removed sending console input. I still left most of the plumbing for it in place though so it should be quite easy to add sending console input to the backend in the future.